db/migrations/20190807230019-add-indexes.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
```

Comment:
Minor: I just noticed the generator adds this while making my own. Since we're using Node 8.x, we're always running in strict mode so [we don't need this](https://stackoverflow.com/questions/31685262/not-recommended-to-use-use-strict-in-es6).

That said, it doesn't hurt.